jwtutil/jwt.go

```diff
@@ -0,0 +1,60 @@
+// Copyright 2022 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package jwtutil provides mechanisms for interacting with JWTs.
+package jwtutil
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/lestrrat-go/jwx/v2/jwk"
+	"github.com/lestrrat-go/jwx/v2/jws"
+	"github.com/lestrrat-go/jwx/v2/jwt"
+)
+
+// JWTVerifier allows for getting public JWK keys from an endpoint and validating JWTs with
+// those keys.
+type JWTVerifier struct {
```

Comment:
Nit: JWT will now be redundant. Something like `jwtutil.Verifier` could be self-explanatory.